appveyor.yml

```diff
@@ -7,14 +7,20 @@ environment:
   APPVEYOR_SAVE_CACHE_ON_ERROR: true
   CLCACHE_SERVER: 1
   PACKAGES: boost-filesystem boost-signals2 boost-interprocess boost-test libevent openssl zeromq berkeleydb secp256k1 leveldb
+  PYTHONIOENCODING: utf-8
 cache:
 - C:\tools\vcpkg\installed
 - C:\Users\appveyor\clcache
 - build_msvc\cache
 init:
 - cmd: set PATH=C:\Python36-x64;C:\Python36-x64\Scripts;%PATH%
+- cmd: set BITCOINPATH=%APPVEYOR_BUILD_FOLDER%\build_msvc\%PLATFORM%\%CONFIGURATION%
+- cmd: set BITCOIND=%BITCOINPATH%\bitcoind.exe
+- cmd: set BITCOINCLI=%BITCOINPATH%\bitcoin-cli.exe
```

Comment:
Would be better to not mix and use powershell.

Reply:
Any reason why that is better?